tutorials/multi-modal/tracing_with_images_tutorial.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<p>Need to prompt for an openai key</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4441/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>